packages/block-library/src/navigation/edit/navigation-menu-selector.js

```diff
@@ -96,26 +79,40 @@ function NavigationMenuSelector( {
 	const menuUnavailable =
 		hasResolvedNavigationMenus && currentMenuId === null;
 
-	useEffect( () => {
-		if ( ! hasResolvedNavigationMenus ) {
-			setSelectorLabel( __( 'Loading …' ) );
-		} else if ( noMenuSelected || noBlockMenus || menuUnavailable ) {
-			setSelectorLabel( __( 'Select menu' ) );
-			setEnableOptions( shouldEnableMenuSelector );
-		}
+	let selectorLabel = '';
 
+	if ( isCreatingMenu || isResolvingNavigationMenus ) {
+		selectorLabel = __( 'Loading …' );
+	} else if ( noMenuSelected || noBlockMenus || menuUnavailable ) {
```

Comment:
`noBlockMenus || menuUnavailable`

These conditions confuse me. I copied when from the setState conditional in `trunk` but why do we want to show a `Select menu` option when there are no block menus or they are unavailable?

Reply:
There may be classic menus to import in that dropdown?

Reply:
Perfect answer. That one will need to go in a comment.